apis/fluentd/v1alpha1/plugins/input/sample.go

```diff
@@ -0,0 +1,15 @@
+package input
+
+// The in_sample input plugin generates sample events. It is useful for testing, debugging, benchmarking and getting started with Fluentd.
+type Sample struct {
+	// The tag of the event. The value is the tag assigned to the generated events.
+	Tag *string `json:"tag,omitempty"`
+	// The number of events in the event stream of each emit.
+	Size *int64 `json:"size,omitempty"`
+	// It configures how many events to generate per second.
+	Rate *int64 `json:"rate,omitempty"`
+	// If specified, each generated event has an auto-incremented key field.
+	AutoIncrementKey *string `json:"auto_increment_key,omitempty"`
```

Comment:
In Fluent Operator, we follow `camelCase` for field name
```suggestion
	AutoIncrementKey *string `json:"autoIncrementKey,omitempty"`
```

Reply:
Thanks for review, fixed as suggested.